addons/mass_mailing/static/src/js/mass_mailing_editor.js

```diff
@@ -90,7 +91,7 @@ options.registry["width-x"] = options.Class.extend({
         var self = this;
         _.each($el.siblings(),function(sib){
             max_width +=  self.get_int_width($(sib));
-        })
+        });
```

Comment:
"Syntax" errors are not to be fixed in stable versions. The only thing you risk by doing that is breaking forward-ports and so the fix itself for recent versions :wink: